src/test/java/org/json/junit/XMLTest.java

```diff
@@ -1247,6 +1249,39 @@ public void testIndentComplicatedJsonObjectWithArrayAndWithConfig(){
+
+        Throwable throwable = assertThrows(JSONException.class, new ThrowingRunnable() {
```

Comment:
It would be more consistent to use the same structure for the validation as the other existing tests in the class, like this:
```
        try {
            XML.toJSONObject(xmlStr);
            fail("Expecting a JSONException");
        } catch (JSONException e) {
            assertEquals("Expecting an exception message",
                    "Misshaped meta tag at 214 [character 12 line 7]",
                    e.getMessage());
        }
```